morphct/system.py

```diff
@@ -86,6 +86,7 @@ def __init__(
             os.makedirs(outpath)
         self.outpath = outpath
         self._chromophores = []
+        self.carrier_data = None
```

Comment:
```suggestion
        self._carrier_data = None
```
if we're going to use the "private" attribute (i.e., with a leading underscore) later, let's keep consistent and use it everywhere